src/assisted_test_infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -624,6 +631,7 @@ def set_ipxe_url(self, network_name: str, ipxe_url: str):
         bootp.setAttribute("file", ipxe_url)
         dhcp_element.appendChild(bootp)
         self.destroy_network(network)
+        self.undefine_network(network)
```

Comment:
where is this been used?

Reply:
When we override the network config with "bootp file" , sometimes it did not work once creating new one.
The undefine removes it totally and we create new one.
The idea here is that we take existing network and adding bootp support (path to the iPXE) .

Reply:
if no objection i will mark it as resolved.